x-pack/plugins/enterprise_search/public/applications/app_search/components/search_ui/search_ui.tsx

```diff
@@ -15,8 +17,14 @@ import { SetAppSearchChrome as SetPageChrome } from '../../../shared/kibana_chro
 import { getEngineBreadcrumbs } from '../engine';
 
 import { SEARCH_UI_TITLE } from './constants';
+import { SearchUILogic } from './search_ui_logic';
 
 export const SearchUI: React.FC = () => {
+  const { initializeData } = useActions(SearchUILogic);
+  useEffect(() => {
```

Comment:
[optional] newline nit
```suggestion
  const { initializeData } = useActions(SearchUILogic);

  useEffect(() => {
```